composer.json

```diff
@@ -30,13 +30,13 @@
     },
 
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^5.5 || ^5.6 || ^7.0",
         "graze/sort": "^2.0.1"
     },
 
     "require-dev": {
         "graze/hamcrest-test-listener": "^1.0",
         "hamcrest/hamcrest-php": "^1.2",
-        "phpunit/phpunit": "^5.0"
+        "phpunit/phpunit": "^4.8"
```

Comment:
If this is going ahead, use `^4.8 || ^5.0`. That should allow people not using PHP 5.5 to install the latest phpunit I think.